manager/gui/src/components/alerts/LrAlertCommentReplies.vue

```diff
@@ -0,0 +1,117 @@
+<!--
```

Comment:
there are multiple console errors and warning while opening show more replies.

Reply:
It was due to the duplicate keys. Modified the Ids in the dummy data.